integration-tests/src/test/java/io/druid/tests/indexer/ITKafkaTest.java

```diff
@@ -189,7 +189,7 @@ public void testKafka()
           event_template,
           event_fmt.print(dt), num_events, 0, num_events
       );
-      LOG.debug("event: [%s]", event);
+      LOG.info("event: [%s]", event);
```

Comment:
@drcrallen Done. I actually removed the prior LOG.info since timestamp info is already included in the event.